packages/backend/src/serviceConfigurations/agentService/resources/normalizedAgent/index.js

```diff
@@ -24,6 +24,22 @@ module.exports = {
     migrations,
     name: 'normalizedAgent',
     relations: ['normalizedAgent'],
+    relationships: {
```

Comment:
it becomes much cleaner in the models repo, but it needs to be documented the difference between `relations` and `relationships` in the resourceConfiguration :-)